welcome/index.adoc

```diff
@@ -42,6 +42,12 @@ To navigate the {product-title} documentation, use the left navigation bar.
 For documentation that is not specific to {product-title}, see the link:https://docs.openshift.com/container-platform/latest/welcome/index.html[OpenShift Container Platform documentation].
 endif::[]
 
+ifdef::microshift[]
+To navigate the {product-title} documentation, use the left navigation bar.
+
+For documentation that is not specific to {product-title}, see the link:https://docs.openshift.com/container-platform/latest/welcome/index.html[OpenShift Container Platform documentation].
```

Comment:
This takes me to the 4.12 docs. Not sure if it is because 4.13 isn't live yet though.

Reply:
I agree, it's just generic text.